### PR TITLE
Release 1.0.0

### PR DESCRIPTION
**Pull request by maxspahn**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -130,3 +130,4 @@ dmypy.json
 # Pyre type checker
 .pyre/
 tempResults/
+experiments
```
Comment (c-salmi) [inline .gitignore:133]: Why should experiments folder be added to ignore? Since it's renamed to examples

**Review by c-salmi — APPROVED** [review]
Looks good! I tried all the fabric and pdplanner examples (forcespro I couldn't because of license). They all ran and were stable, only the planar_arm n3 for fabrics might need some tuning, but I experienced no errors, so I'll aprove it :+1:

File: .gitignore (hunk 1 above)
Comment (maxspahn) [reply .gitignore:133]: It was not a pure renaming. I still have the experiments locally, but I didn't want them in the repo any longer.

**Comment by c-salmi** [comment]
All the acados parts are now integrated. From my side it can be merged right now.

**Pull request merged**